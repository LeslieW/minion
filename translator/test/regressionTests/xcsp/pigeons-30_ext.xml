<?xml version="1.0" encoding="UTF-8"?>

<instance>
<presentation name="?" maxConstraintArity="30" format="XCSP 2.0"/>

<domains nbDomains="1">
<domain name="D0" nbValues="29">0..28</domain>
</domains>

<variables nbVariables="30">
<variable name="V0" domain="D0"/>
<variable name="V1" domain="D0"/>
<variable name="V2" domain="D0"/>
<variable name="V3" domain="D0"/>
<variable name="V4" domain="D0"/>
<variable name="V5" domain="D0"/>
<variable name="V6" domain="D0"/>
<variable name="V7" domain="D0"/>
<variable name="V8" domain="D0"/>
<variable name="V9" domain="D0"/>
<variable name="V10" domain="D0"/>
<variable name="V11" domain="D0"/>
<variable name="V12" domain="D0"/>
<variable name="V13" domain="D0"/>
<variable name="V14" domain="D0"/>
<variable name="V15" domain="D0"/>
<variable name="V16" domain="D0"/>
<variable name="V17" domain="D0"/>
<variable name="V18" domain="D0"/>
<variable name="V19" domain="D0"/>
<variable name="V20" domain="D0"/>
<variable name="V21" domain="D0"/>
<variable name="V22" domain="D0"/>
<variable name="V23" domain="D0"/>
<variable name="V24" domain="D0"/>
<variable name="V25" domain="D0"/>
<variable name="V26" domain="D0"/>
<variable name="V27" domain="D0"/>
<variable name="V28" domain="D0"/>
<variable name="V29" domain="D0"/>
</variables>

<constraints nbConstraints="1">
<constraint name="C0" arity="30" scope="V0 V1 V2 V3 V4 V5 V6 V7 V8 V9 V10 V11 V12 V13 V14 V15 V16 V17 V18 V19 V20 V21 V22 V23 V24 V25 V26 V27 V28 V29" reference="global:allDifferent"/>
</constraints>
</instance>
