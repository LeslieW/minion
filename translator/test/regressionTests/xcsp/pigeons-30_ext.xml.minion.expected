MINION 3

#This file has been automatically generated by TAILOR v0.2
#bug-reports: andrea@cs.st-and.ac.uk

# amount of common subexpressions used:0
# amount of inferred common subexpressions used:0
# amount of original variables saved (over direct equality):0
# amount of constraints:1
# amount of auxiliary variables: 0

**VARIABLES**
DISCRETE V0 {0..28}
DISCRETE V1 {0..28}
DISCRETE V2 {0..28}
DISCRETE V3 {0..28}
DISCRETE V4 {0..28}
DISCRETE V5 {0..28}
DISCRETE V6 {0..28}
DISCRETE V7 {0..28}
DISCRETE V8 {0..28}
DISCRETE V9 {0..28}
DISCRETE V10 {0..28}
DISCRETE V11 {0..28}
DISCRETE V12 {0..28}
DISCRETE V13 {0..28}
DISCRETE V14 {0..28}
DISCRETE V15 {0..28}
DISCRETE V16 {0..28}
DISCRETE V17 {0..28}
DISCRETE V18 {0..28}
DISCRETE V19 {0..28}
DISCRETE V20 {0..28}
DISCRETE V21 {0..28}
DISCRETE V22 {0..28}
DISCRETE V23 {0..28}
DISCRETE V24 {0..28}
DISCRETE V25 {0..28}
DISCRETE V26 {0..28}
DISCRETE V27 {0..28}
DISCRETE V28 {0..28}
DISCRETE V29 {0..28}

# aliases

**SEARCH**

PRINT [
[V0],[V1],[V2],[V3],[V4],[V5],[V6],[V7],
[V8],[V9],[V10],[V11],[V12],[V13],[V14],[V15],
[V16],[V17],[V18],[V19],[V20],[V21],[V22],[V23],
[V24],[V25],[V26],[V27],[V28],[V29]]

VARORDER [
V0,V1,V2,V3,V4,V5,V6,V7,
V8,V9,V10,V11,V12,V13,V14,V15,
V16,V17,V18,V19,V20,V21,V22,V23,
V24,V25,V26,V27,V28,V29]


**CONSTRAINTS**

alldiff([ V0, V1, V2, V3, V4, V5, V6, V7, V8, V9, V10, V11, V12, V13, V14, V15, V16, V17, V18, V19, V20, V21, V22, V23, V24, V25, V26, V27, V28, V29])

**EOF**
